src/MessageFormats/CloudEvent.cs

```diff
@@ -0,0 +1,13 @@
+﻿using System;
+namespace Likvido.Worker.AzureStorageQueue.MessageFormats
+{
+    public class CloudEvent<TMessage>
+    {
+        public string Id { get; set; } = null!;
+        public string Source { get; set; } = null!;
+        public TMessage Data { get; set; } = default!;

```

Comment:
According to the spec, this is optional 
https://github.com/cloudevents/spec/blob/v1.0/spec.md#event-data